main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
```

Comment:
Awesome job! Maybe consider making a function, this way you can easily call the function whenever you need to use it :)